app/controllers/carts_controller.rb

```diff
@@ -18,11 +20,21 @@ def destroy
     redirect_to carts_path
   end
 
-  def check_food_status(food, qty, food_id)
+  def check_food_status(food, qty, food_id, prep_time)
     unless food.status == "available"
       @ordered_foods.delete(food_id)
     else
       @total += (food.price * qty)
+      @prep_total += (food.prep_time * qty)
+      require "pry-nav"; binding.pry
```

Comment:
Remove debugger entry point `binding.pry`.<br>Do not use semicolons to terminate expressions.